lib/pf/cmd/pf/generatemariadbconfig.pm

```diff
@@ -37,6 +38,13 @@ use pf::config qw(
     $management_network
     $DISTRIB
 );
+use pf::constants::eventLogger;
```

Comment:
Are the table we log events for configurable ? Or the moment you enable the feature, it installs the triggers for all the tables ?

Reply:
It is configurable the following tables are supported admin_api_audit_log,  auth_log, dhcp_option82, dns_audit_log, radius_audit_log.

Reply:
I guess I'm missing something in the code because right now what I see is that the trigger will be installed for all the tables and not just the tables that are part of the configured namespaces

Reply:
Check the logic here https://github.com/inverse-inc/packetfence/blob/c92364112efb1f5948604dd6855207eaca3b88d5/conf/mariadb/mariadb.sql.tt.example to understand better.
The basic logic is the following.
Re add the pf_logger function.
Drop all triggers.
If trigger is configured then add it.

Reply:
You look at this test to understand a little more.
 https://github.com/inverse-inc/packetfence/blob/bc841ff6a58f2268f5c383c63f950c3426693f2b/t/unittest/cmd/pf/generatemariadbconfig.t